django/db/migrations/writer.py

```diff
@@ -184,7 +184,15 @@ def as_string(self):
         if self.migration.replaces:
             items['replaces_str'] = "\n    replaces = %s\n" % self.serialize(self.migration.replaces)[0]
 
-        return (MIGRATION_TEMPLATE % items).encode("utf8")
+        prepared_migration_statement = (MIGRATION_TEMPLATE % items).encode("utf8")
+        try:
+            import autopep8
+            return autopep8.fix_code(prepared_migration_statement,
+                options=autopep8.parse_args(['--aggressive', '']))
+        except ImportError:
+            pass
+        finally:
```

Comment:
Interesting. I just tried it. This returns 2.

    def func():
        try:
            return 1
        finally:
            return 2